CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [v1.7.0] - Unreleased Phase 6
 
 ### Added
+- Created CLI [Applet](https://github.com/signnow/SignNowNodeSDK/blob/master/bin/duplicate-template.js) 
```

Comment:
would be better to write like: Documented feature ... See examples... New style that we accepted